collectors/0/samza_staging.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+
+import json
+import re
+import sys
+
+from kafka import KafkaConsumer
+
+from collectors.lib import utils
+
+ENV = 'staging'
+
+def main(argv):
+
+    utils.drop_privileges()
+
+    # Pulls metrics from the "samza_metrics" Kafka topic
+    consumer = KafkaConsumer('samza_metrics',
+                           group_id='samza_metrics',
+                           bootstrap_servers=['kafkastaging-1b-east-fd8d3028-resonantdromedary.us-east-1.optimizely:9092',
+                                              'kafkastaging-1e-east-0a3cafa9-enchantedrhinoceros.us-east-1.optimizely'])
```

Comment:
hmm. maybe take it from parameterize through environment variables, and set them in tcollector runit service in chef. i agree it's a hassle, so i'm ok if you decide to just hardcode them in the beginning of the file as constants.